Public/config.ps1

```diff
@@ -44,6 +42,24 @@ $artConfig = [PSCustomObject]@{
 
 }
 
+# [optional] log filename configuration
+# For log filenames to update with the current time when they are accessed, they need to be initiated as ScriptProperty properties
+$scriptParam = @{
+  MemberType  = "ScriptProperty"
+  InputObject = $PrivateConfig
```

Comment:
@matt-kowalski can you please change it from `$PrivateConfig` to `$artConfig` in both the places ?